sonar-plugin-api/src/main/java/org/sonar/api/web/WidgetScope.java

```diff
@@ -32,7 +32,10 @@
  * then is is assumed project scoped.
  *
  * @since 3.1
+/**
+ * @deprecated since 6.2, as dashboards have been removed
  */
+@Deprecated
 @Retention(RetentionPolicy.RUNTIME)
 @Target(ElementType.TYPE)
 public @interface WidgetScope {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1133)